types/index.test-d.ts

```diff
@@ -129,20 +138,24 @@ const appWithHttp2 = fastify({ http2: true })
 
 appWithHttp2
   .register(fastifyStatic, options)
-  .after(() => {
-    appWithHttp2.get('/', (request, reply) => {
+  .after((err, instance) => {
```

Comment:
This is a breaking change since we no longer support the scenario of the test: if `err` and `instance` params are not defined, `.sendFile` will not exist at TS level
![image](https://github.com/user-attachments/assets/614bd2f0-4c19-4167-b96b-915462df0eff)

Reply:
I do not see which test case this is, the code is obscured with the error message. Can you provide a test case that i can take a look at?